.github/workflows/main.yml

```
@@ -64,6 +64,38 @@ jobs:
         # so we instruct git-diff to ignore those when considering changes.
         git --no-pager diff --ignore-cr-at-eol --exit-code
 
+  build-publish-website:
+    name: Check for license header comments on python files
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2.3.1
+      with:
+        persist-credentials: false
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+      run: sudo apt -y install build-essential libfreetype-dev unixodbc-dev hugo
```

```suggestion
      run: sudo apt-get -y install build-essential libfreetype-dev unixodbc-dev hugo
```
as discussed elsewhere